internal/test/random.go

```diff
@@ -9,7 +9,7 @@ var letters = []rune("abcdefghijklmnopqrstuvwxyz")
 func GenerateName() string {
 	b := make([]rune, 5)
 	for i := range b {
-		b[i] = letters[rand.Intn(len(letters))]
+		b[i] = letters[rand.Intn(len(letters))] //nolint:gosec
```

Comment:
`math/rand` is just fine here since this ain't secure use case but just test data.